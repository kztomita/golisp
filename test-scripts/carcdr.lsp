(print (car '(1 2 3)))
(print (car '()))
(print (car nil))
(print (cdr '(1 2 3)))
(print (cdr '()))
(print (cdr nil))


