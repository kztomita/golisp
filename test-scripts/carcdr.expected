
1 
nil 
nil 
(2 3) 
nil 
nil 
